elasticgraph-indexer/lib/elastic_graph/indexer/datastore_indexing_router.rb

```diff
@@ -195,6 +195,8 @@ def source_event_versions_in_index(operations)
         end
 
         client_names_and_results = Support::Threading.parallel_map(ops_by_client_name) do |(client_name, all_ops)|
+          # @type block: [String, Symbol, Array[untyped] | Hash[_Operation, Array[Integer]]]
```

Comment:
Keeping this change here because this is related a bug that is fixed in 1.10.0. https://github.com/soutaro/steep/pull/1534
(I found the problem with this line!)